src/ossf_scorecard/__init__.py

```diff
@@ -0,0 +1,8 @@
+#
```

Comment:
We need to have this notice at every file.

Reply:
done